Tests/BugsnagSinkTests.m

```diff
@@ -299,12 +299,11 @@ - (void)testEventApp {
     NSDictionary *event = [self.processedData[@"events"] firstObject];
     NSDictionary *app = event[@"app"];
     XCTAssertNotNil(app);
-    XCTAssertEqual(11, app.count);
+    XCTAssertEqual(9, app.count);
     
     XCTAssertEqualObjects(app[@"id"], @"net.hockeyapp.CrashProbeiOS");
     XCTAssertNotNil(app[@"type"]);
     XCTAssertEqualObjects(app[@"version"], @"1.0");
-    XCTAssertEqualObjects(app[@"name"], @"CrashProbeiOS");
     XCTAssertEqualObjects(app[@"bundleVersion"], @"1");
     XCTAssertEqualObjects(app[@"releaseStage"], @"production");
     XCTAssertEqualObjects(app[@"dsymUUIDs"], @[@"D0A41830-4FD2-3B02-A23B-0741AD4C7F52"]);
```

Comment:
This UUID could live in `BugsnagTestConstants.h`

Reply:
IMO it's best to keep this ID as-is for now, since it's only used in one test source file and it's unrelated to this changeset. Can update if you feel strongly about this